### ISSUE TITLE
Forbid double+ assignment targets

### ISSUE THREAD

**Issue by sobolevn** [wemake-services/wemake-python-styleguide#400]
# Rule request

## Thesis

We should not allow things like: 

1. `some1 = some20, some21 = some3`
2. `some1 = some2 = some3 = some4`

## Reasoning

Declaring multiple variables this way leaves more questions than answers.
Why all these variables are equal?
Why do you need so many variables at the same time?

The solution is to write:

```python
some1 = some3
some2 = some3
```
